app/bundles/LeadBundle/Model/FieldModel.php

```diff
@@ -32,113 +32,242 @@ class FieldModel extends FormModel
         ],
         'phone'            => [
             'type'     => 'tel',
             'fixed'    => true,
             'listable' => true,
+            'object'   => 'lead'
         ],
         'fax'              => [
             'type'     => 'tel',
             'listable' => true,
+            'object'   => 'lead'
         ],
         'address1'         => [
             'fixed'    => true,
             'listable' => true,
+            'object'   => 'lead'
         ],
         'address2'         => [
             'fixed'    => true,
             'listable' => true,
+            'object'   => 'lead'
         ],
         'city'             => [
             'fixed' => true,
+            'object'=> 'lead'
         ],
         'state'            => [
             'type'  => 'region',
             'fixed' => true,
+            'object'=> 'lead'
         ],
         'zipcode'          => [
             'fixed' => true,
+            'object'=> 'lead'
         ],
         'country'          => [
             'type'  => 'country',
             'fixed' => true,
+            'object'=> 'lead'
         ],
         'preferred_locale' => [
             'type'     => 'locale',
             'fixed'    => true,
             'listable' => true,
+            'object'   => 'lead'
         ],
         'attribution_date' => [
             'type'     => 'datetime',
             'fixed'    => true,
             'listable' => true,
+            'object'   => 'lead'
         ],
         'attribution'      => [
             'type'       => 'number',
             'properties' => ['roundmode' => 4, 'precision' => 2],
             'fixed'      => true,
             'listable'   => true,
+            'object'     => 'lead'
         ],
         'website'          => [
             'type'     => 'url',
             'listable' => true,
+            'object'   => 'lead'
         ],
         'facebook'   => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
         'foursquare' => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
         'googleplus' => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
         'instagram'  => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
         'linkedin'   => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
         'skype'      => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
         'twitter'    => [
             'listable' => true,
             'group' => 'social',
+            'object'=> 'lead'
         ],
     ];
 
+    static public $coreCompanyFields   = [
+        // Listed according to $order for installation
+        'companyNumber'     => [
```

Comment:
Let's keep these consistent with lead fields - instead of camel case, use `company_number` - same for the rest of the company fields